compiler/rustc_span/src/hygiene.rs

```diff
@@ -1484,44 +1471,38 @@ impl<D: Decoder> Decodable<D> for SyntaxContext {
 #[instrument(level = "trace", skip(ctx), ret)]
 fn update_disambiguator(
     expn_data: &mut ExpnData,
-    dep_node: FakeDepNode,
+    hash_extra: impl Hash + Copy + fmt::Debug,
```

Comment:
```suggestion
    hash_extra: impl Hash + Copy
```
Unused bound (in a couple of other places too).